htstress.c

```diff
@@ -65,6 +65,13 @@ typedef void (*sighandler_t)(int);
 
 #define MAX_EVENTS 256
 
+/**
+ * struct econn - represents one connection to server from htstress.
+ * @fd: clientfd
+ * @offs: the buffer offset represents data already read from or sent to buffer.
+ * @flags: flags represents the http response status from server, only used with
+ * BAD_REQUEST.
```

Comment:
Indent this line. It should be aligned with "flags ".

Reply:
fixed in https://github.com/sysprog21/khttpd/pull/8/commits/be11e24a0e094e21f849d5e28b393a9f9df55cfc